SPARQLWrapper/main.py

```diff
@@ -136,18 +135,27 @@ def main(test=None):
     sparql.setReturnFormat(args.format)
     results = sparql.query().convert()
 
-    if args.format == "json":
+    ct = type(results)
+    if ct is dict:
```

Comment:
I think it may be better to use `isinstance` in all cases:

```suggestion
    if isinstance(result, dict):
```